cache/disk/lru.go

```diff
@@ -67,6 +69,10 @@ func NewSizedLRU(maxSize int64, onEvict EvictCallback) SizedLRU {
 		cache:   make(map[interface{}]*list.Element),
 		onEvict: onEvict,
 
+		gaugeCacheLastEvictAtime: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "bazel_remote_disk_cache_last_evicted_atime",
+			Help: "The access time of the most recently evicted item from the LRU cache.",
```

Comment:
I wonder if a better metric would be the amount of time since the last evicted item was last touched? ie "now - atime of evicted item" - that way a graph of the metric over time would be a flat line when things are fairly consistent, instead of a line with a constant positive gradient.